src/main/java/net/jingles/enchantments/Commands.java

```diff
@@ -27,124 +27,148 @@
 
-      ItemMeta meta = held.getItemMeta();
+        meta = held.getItemMeta();
 
-      if (meta == null || !meta.hasEnchant(enchant)) {
-        sender.sendMessage(ERROR + "This tem does not have that enchantment!");
-        return false;
-      }
+        if (meta == null || !meta.hasEnchant(enchant)) {
+          sender.sendMessage(ERROR + "This item does not have that enchantment!");
+          return false;
+        }
 
-      meta.removeEnchant(enchant);
+        meta.removeEnchant(enchant);
 
-      if (meta.getLore() != null) {
-        List<String> lore = meta.getLore();
-        lore.removeIf(line -> line.contains(enchant.getName()));
-        meta.setLore(lore);
-      }
+        if (meta.getLore() != null) {
+          lore = meta.getLore();
+          lore.removeIf(line -> line.contains(enchant.getName()));
+          meta.setLore(lore);
+        }
 
-      held.setItemMeta(meta);
-      sender.sendMessage(TITLE + "Successfully removed enchantment.");
+        held.setItemMeta(meta);
+        sender.sendMessage(TITLE + "Successfully removed enchantment.");
+        break;
 
-    } else if (args[0].equalsIgnoreCase("list")) {
+      case "list":
 
-      String list = ChatColor.AQUA + Enchantments.REGISTERED.stream()
-              .map(CustomEnchant::getName).collect(Collectors.joining(", "));
-      sender.sendMessage(TITLE + "Registered custom enchantments: " + list);
+        String list = ChatColor.AQUA + Enchantments.REGISTERED.stream()
+                .map(CustomEnchant::getName).collect(Collectors.joining(", "));
+        sender.sendMessage(TITLE + "Registered custom enchantments: " + list);
+        break;
 
-    }
+      case "info":
+
+        if (args.length < 2) {
+          sender.sendMessage(ERROR + "Incorrect arguments: /enchantments info <enchantment name>");
+          return false;
+        }
+
+        try {
+        enchant = getCustomEnchant(Arrays.copyOfRange(args, 1, args.length));
+        sender.sendMessage(TITLE + enchant.getDescription());
+      } catch (NullPointerException | IllegalArgumentException e) {
+          sender.sendMessage(ERROR + "An enchantment with that name could not be found.");
+        }
+        break;
+
+      case "help":
+        sender.sendMessage(TITLE + ChatColor.GOLD + "Available Commands:");
+        sender.sendMessage(ChatColor.GRAY + "/enchantments enchant <level> <enchantment name>");
+        sender.sendMessage(ChatColor.GRAY + "/enchantments unenchant <enchantment name>");
+        sender.sendMessage( ChatColor.GRAY + "/enchantments info <enchantment name>");
+        sender.sendMessage(ChatColor.GRAY + "/enchantments list");
+        break;
+
+      default:
+        sender.sendMessage(ERROR + "Incorrect arguments.");
+        return false;
 
+    }
     return true;
   }
 
   private ItemStack getHeldItem(CommandSender sender) {
     if (!(sender instanceof Player)) {
       sender.sendMessage(ERROR + "Only players can use this command!");
+      return null;
```

Comment:
Returning here is unnecessary. Without it, it would just skip the else statement... which leads straight to return null anyway